### DISCUSSION TITLE
[Unstable] Testing an outpost module in sub editor loads a whole outpost

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14265] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If I test an outpost module it loads an entire outpost?

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/81bc5010-2c1a-4651-9319-9f44d1df06b7)
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/798a34ba-f44f-4911-9d1e-273027b20614)

### Reproduction steps

1. Load an outpost module in sub editor
2. Test

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/2a4f925e-01d4-44c1-ba82-1e1786e8363b)

If this is intended please undo it.

**Comment by Regalis11** [comment]
This is intentional. Why do you consider it a bad thing or something that should be reverted? We added it because it makes it much easier to test that the module works in an actual outpost, appears in the intended outpost type, connects to other modules and the power grid correctly, etc. If the module can't be included in any type of outpost if it's for example not included in any type of outpost generation parameters, it should just load without the rest of the outpost though.

**Reply by ubertpendragon** [reply]
The big sore point of that change is that it will pick a spawnpoint at random and potentially spawn you at the other side of the outpost so you can't test your specific module properly (and you also wont have ID tags).

I guess if you fix this somehow then it would be fine

**Comment by Rokvach** [comment]
Fixed and fix tested to be working correctly. Fix coming in the next Unstable build. Closing.
